lib/apiservers/service/swagger.json

```diff
@@ -0,0 +1,879 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "vic-machine API",
+    "description": "An API for interacting with vic-machine as a RESTful web service.",
+    "license": {
+      "name": "Apache 2.0",
+      "url": "https://raw.githubusercontent.com/vmware/vic/master/LICENSE"
+    },
+    "version": "v0.1.0"
+  },
+  "basePath": "/container",
+  "schemes": [
+    "http"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/": {
+      "get": {
+        "summary": "Show VIC metadata",
+        "description": "A `GET` request on the base resource will return a JSON object containing metadata including the version number of the service software and a list of known appliance ISOs.",
+        "responses": {
+          "200": { "$ref": "#/responses/metadata" }
+        },
+        "security": []
+      }
+    },
+    "/version": {
+      "get": {
+        "summary": "Show VIC version information",
+        "description": "A `GET` request on the `version` sub-resource will return the version number of the service software.",
+        "produces": [
+          "text/plain"
+        ],
+        "responses": {
+          "200": { "$ref": "#/responses/version" }
+        },
+        "security": []
+      }
+    },
+    "/{target}": {
```

Comment:
Should this be `/targets/{target}`? This would reduce the chances of target name colliding with other resource names under `/`.

Reply:
Because `{target}` must be a valid IP address or hostname, I think the risk of collision is low.

Given the current `vic-machine` operations, and general model that all VIC operations involve a vSphere resource of some sort, `/version` may be the _only_ resource directly under `/`.

Reply:
Name collisions aside, it just seems odd to address a target under `/`. We're not managing the target after all. As a consumer of the API, I would expect endpoints under `/` to be directly relevant to the vic-machine service itself, like `/version`.